moveit_msgs/JointConstraint[] constraints
float64 orientation
float64 position
---
string error
